app/views/todos/todo.erb

```diff
@@ -0,0 +1,32 @@
+<center><h1> The to do list: </h1>
+
+<table>
+ <% @todo.each do|t| %>
+  <tr>
+    <td>
+      <form action ="complete_todo">
+        <input type="checkbox" name = "completed">
+      </form>
+    </td>
+    <td>
+      <a href="/todo_page/<%=t.id%>">
+      <%= t.title %>
+      </a>
+    </td>
+    <td>
+      <form action = "/edit_todo_form/<%=t.id%>">
```

Comment:
This is Tray. We decided to use a form in this instance because I thought that was the only way to make a button work as a link?